src/sharedHooks/useLocalObservations.js

```diff
@@ -31,7 +31,9 @@ const useLocalObservations = ( ): Object => {
         [["needs_sync", true], ["_created_at", true]]
       );
 
-      const obsNotFlaggedForDeletion = sortedCollection.filtered( "_deleted_at == nil" );
+      // eslint-disable-next-line max-len
+      const deletionFilters = "_deleted_at == nil OR _pending_deletion == false OR _pending_deletion == null";
```

Comment:
The "null" needs to be "nil" here, right?
Also, does the order matter for the realm ORs, should == nil be before == false, or it doesn't matter?

Reply:
good call on nil. i believe filtering in Realm with ORs is purely filtering and will remain in the same order as the code above:
```
const sortedCollection = collection.sorted(
        [["needs_sync", true], ["_created_at", true]]
      );
```